src/C4StartupScenSelDlg.cpp

```diff
@@ -1400,6 +1400,21 @@ void C4StartupScenSelDlg::OnShown()
 	pScenLoader->Load(StdStrBuf(Config.General.ExePath));
 	UpdateList();
 	UpdateSelection();
+#ifdef __linux__
+	StdStrBuf data_folder(getenv("XDG_CONFIG_DATA"));
+	if (!data_folder.getSize()){
+	    data_folder = getenv("HOME");
+	    data_folder += "/.local/share/legacyclonk";
+	}
+    data_folder += "/";
+	pScenLoader->Load(data_folder);
+	UpdateList();
+	UpdateSelection();
+	data_folder = "/usr/share/legacyclonk/";
+	pScenLoader->Load(data_folder);
+	UpdateList();
+	UpdateSelection();
+#endif
```

Comment:
That's a lot of duplicate code for no good reason. I also don't like that scenarios should be in the **configuration** directory - what?

Reply:
Using XDG_CONFIG_DATA here instead of XDG_DATA_HOME here is a brain issue on my side - the fallback code actually loads scenarios from the (proper) local data directory.